File: server2/src/main/java/com/wepay/business/model/Good.java
package com.wepay.business.model;

import javax.persistence.*;


@Entity
@Table(name = "good")
public class Good {

  @Id
  @GeneratedValue(strategy = GenerationType.AUTO)
  private long id;

  @Column(name = "name")
  private String name;

  @Column(name = "price")
  private double price;

  @Column(name = "img")
  private String img;

  @Column(name = "info")
  private String info;

  @Column(name = "amount")
  private int amount;

  @Column(name = "address")
  private  String address;

  @Column(name = "soldAmount")
  private String soldAmount;

  @Column(name = "sellerId")
  private String sellerId;

  public Good(){

  }

  public Good(String name, Double price, String info, int amount) {
    this.name = name;
    this.price = price;
    this.info = info;
    this.amount = amount;
  }

  public long getId() {
    return id;
  }

  public void setId(long id) {
    this.id = id;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public double getPrice() {
    return price;
  }

  public void setPrice(double price) {
    this.price = price;
  }

  public String getImg() {
    return img;
  }

  public void setImg(String img) {
    this.img = img;
  }

  public String getInfo() {
    return info;
  }

  public void setInfo(String info) {
    this.info = info;
  }

  public int getAmount() {
    return amount;
  }

  public void setAmount(int amount) {
    this.amount = amount;
  }

  public String getAddress() {
    return address;
  }

  public void setAddress(String address) {
    this.address = address;
  }

  public String getSoldAmount() {
    return soldAmount;
  }

  public void setSoldAmount(String soldAmount) {
    this.soldAmount = soldAmount;
  }

  public String getSellerId() {
    return sellerId;
  }

  public void setSellerId(String sellerId) {
    this.sellerId = sellerId;
  }
}

File: server/schema/good.js
/* jshint indent: 2 */

module.exports = function(sequelize, DataTypes) {
  return sequelize.define('good', {
    'id': {
      type: DataTypes.INTEGER(11),
      allowNull: false,
      primaryKey: true,
      autoIncrement: true
    },
    'name': {
      type: DataTypes.STRING(100),
      allowNull: false
    },
    'price': {
      type: "DOUBLE(10,2)",
      allowNull: false
    },
    'img': {
      type: DataTypes.STRING(255),
      allowNull: true
    },
    'info': {
      type: DataTypes.STRING(255),
      allowNull: true
    },
    'amount': {
      type: DataTypes.INTEGER(255),
      allowNull: true
    },
    'address': {
      type: DataTypes.STRING(255),
      allowNull: true
    },
    'soldAmount': {
      type: DataTypes.INTEGER(255),
      allowNull: true,
      defaultValue: '0'
    },
    'sellerId': {
      type: DataTypes.INTEGER(11),
      allowNull: true,
      references: {
        model: 'seller',
        key: 'sellerId'
      }
    }
  }, {
    tableName: 'good'
  });
};
